src/main/java/com/connexta/search/index/IndexManagerImpl.java

```diff
@@ -47,9 +53,10 @@ public void index(
 
     final Index index;
     try {
-      index = new Index(productId, IOUtils.toString(inputStream, StandardCharsets.UTF_8));
+      JsonNode jsonNode = parseJson(file.getInputStream());
+      index = new Index(productId, getElement(jsonNode, EXT_EXTRACTED_TEXT));
```

Comment:
❓What happens if the json doesn't have a `ext.extracted.text` element? Or the json is empty completely? Or the value for `ext.extracted.text` is an empty string. I think it's worth adding tests for at least these 3 cases.

Reply:
I added a couple parametrized tests to account for this. One of the Indexing tests that we use a Solr mock has started to fail and I can't figure out why. I disabled the test and added a TODO so that we can revisit it later.